applications/ContactStructuralMechanicsApplication/custom_utilities/interface_preprocess.cpp

```diff
@@ -119,12 +127,49 @@ void InterfacePreprocessCondition::GenerateInterfacePart<3>(
 /***********************************************************************************/
 /***********************************************************************************/
 
+std::unordered_map<IndexType, Properties::Pointer> InterfacePreprocessCondition::CreateNewProperties(ModelPart& rOriginPart)
+{
+    const SizeType number_properties = mrMainModelPart.NumberOfProperties();
+
+    std::unordered_map<IndexType, Properties::Pointer> new_properties;
+    // Reorder id Properties
+    IndexType count = 0;
+    for (auto it_prop = mrMainModelPart.PropertiesBegin(); it_prop < mrMainModelPart.PropertiesEnd(); ++it_prop) {
+        ++count;
+        it_prop->SetId(count);
```

Comment:
why do you change the Id? it will not any longer correspond to what you provide as input. Do you REALLY NEED to do this?

Reply:
The idea is to avoid problem creating new properties, it to avoid duplicate Ids

Reply:
Done, @RiccardoRossi  you were right, the problem was that I was using the properties iterator and the array of properties from the main model part was overrided at the same time, provoking a memory error